File: 1.两数之和.go
/*
 * @lc app=leetcode.cn id=1 lang=golang
 *
 * [1] 两数之和
 */

// @lc code=start
func twoSum(nums []int, target int) []int {
	hashmap := make(map[int]int)
	for k, v := range nums {
		if _, ok := hashmap[target-v]; ok {
			return []int{hashmap[target-v], k}
		}
		hashmap[v] = k
	}
	return []int{}
}

// @lc code=end

File: 1.两数之和.py
#
# @lc app=leetcode.cn id=1 lang=python3
#
# [1] 两数之和
#

# @lc code=start
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hashMap = dict()
        for k, v in enumerate(nums):
            if target - v in hashMap:
                return [hashMap[target-v], k]
            hashMap[v] = k
        return []
# @lc code=end
